src/css/component.nav-secondary.css

```diff
@@ -0,0 +1,157 @@
+/* ---------- COMPONENT nav secondary ---------- */
+
+.nav-secondary {
+  --nav-secondary-sub-gap-x: var(--size-4);
+  --nav-secondary-link-color: var(--color-gunmetal);
+  --nav-secondary-link-hover-color: var(--color-red-500);
+  --nav-secondary-link-active-color: var(--color-red-600);
+  margin: var(--size-8) 0;
+}
+
+.nav-secondary--transparent {
+  --nav-secondary-link-color: var(--color-grey-500);
+  --nav-secondary-link-hover-color: var(--color-white);
+  --nav-secondary-link-active-color: var(--color-white);
+}
+
+.nav-secondary a {
+  color: var(--nav-secondary-link-color);
+  display: inline-block;
+  text-decoration: none;
+  transition: color var(--animation-easing) var(--animation-duration);
+}
+
+.nav-secondary a:hover {
+  color: var(--nav-secondary-link-hover-color);
+  text-decoration: underline;
+}
+
+.nav-secondary a:active {
+  color: var(--nav-secondary-link-active-color);
+  text-decoration: underline;
+}
+
+.nav-secondary [aria-current="page"] {
+  color: var(--nav-secondary-link-hover-color);
+}
+
+.nav-secondary__pipe {
+  display: none;
+}
+
+.nav-secondary__top {
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-semibold);
+  line-height: var(--size-5);
+}
+
+.nav-secondary__sub {
+  font-size: var(--font-size-sm);
+  line-height: var(--size-5);
+  margin-bottom: 0;
+}
+
+.js-nav-secondary {
```

Comment:
Option: Might this be `.js .nav-secondary` and leverage the document's `.js` class, as an alternative to creating a new set of `.js-`-prefixed classes?

Reply:
Yeah, good shout. All done.